### ISSUE TITLE
🔎 SEO - Pages not being indexed - Part 2

### ISSUE THREAD

**Issue by Calinator444** [SSWConsulting/SSW.Website#2812]
Based on the email chain:

From: @wicksipedia
Sent: Monday, June 24, 2024 11:20 AM
To: @andrewwaltosssw
Cc: @sethdaily ; SSW Website v3 [SSWWebsiteV3@ssw.com.au](mailto:SSWWebsiteV3@ssw.com.au); @camillars; Jeoffrey Fischer [SSW] [JeoffreyFischer@ssw.com.au](mailto:JeoffreyFischer@ssw.com.au)
Subject: Re: New reasons prevent pages in a sitemap from being indexed on site https://www.ssw.com.au/

#### Description
Search Console has identified that some pages on our website are not being indexed.

#### URL
https://search.google.com/u/1/search-console/index?resource_id=https://www.ssw.com.au/&utm_source=wnc_20237597&utm_medium=gamma&utm_campaign=wnc_20237597&utm_content=msg_110624660&hl=en

#### Solution

- [ ] Soft 404 - 480 pages
- [ ] Blocked by robots.txt - 257 pages
- [x] Server error (5xx) - 89 pages
- [x] Blocked due to other 4xx issue - 1 page
- [ ] Blocked due to access forbidden (403) - 1 page

#### Acceptance Criteria

- [ ] All issues are fixed
- [x] Reply 'Done' to all emails mentioned in this PBI and @mention the sender with 'Done'"

**Comment by Calinator444** [comment]
> Soft 404s 

Some of the `/index.html` pages under `/archive` are not being indexed due to there being little content. As per my conversation with @wicksipedia these pages don't need to be indexed.

In addition some of the old v1 pages have shopping links that no longer work because the shop on the v1 site no longer works. these pages will soft 404.








> Blocked by robots.txt - 257 pages 

Most of these pages are on the v1 site which is being switched off anyway, Only 4 pages on the live site appear here, and they're all 500/404 pages so need to discuss with @wicksipedia whether they should be indexed. I'm assuming they shouldn't be indexed. 

List of pages relevant pages excluded include:
- https://www.ssw.com.au/404-useful-error-page
- https://www.ssw.com.au/404.html
- https://www.ssw.com.au/500
- https://www.ssw.com.au/404

>  Server error (5xx) - 89 pages

Again, this was mostly noise coming from the v1 site. There's a few links to tina template pages in the static content some of the pages on the site being referenced. When Google tries to crawl these links it, of course, returns a 500. Will need to investigate this on Monday.

![Image](https://github.com/SSWConsulting/SSW.Website/assets/65635198/3911a83d-94a3-4191-8af5-39a905368083)

****

>  Blocked due to other 4xx issue - 1 page

It was trying to index a broken link within a rule. I've fixed the link here: https://github.com/SSWConsulting/SSW.Rules.Content/pull/8813


>  Blocked due to access forbidden (403) - 1 page

The broken link points to http://ssw.com.au/ssw/TeamCalendar/Installation/, however there's no default page at that route. Google attempted to index the page because other pages on the site point to that URL.


This page was also in the 403 list: https://www.ssw.com.au/ssw/Redirect/Access/AccessTrial.htm. However it's a redirect which will be gone when v1 server goes offline. Adam already agreed to scrap the /Redirect route.

**Comment by Calinator444** [comment]
TLDR: 

> Soft 404

Not Done - A lot of archived index.html pages here that aren't being indexed due to them having low quality content. Some shopping cart pages from the v1 site also appear here that no longer work.

> Blocked by robots.txt - 257 pages

Not Done - Most of these pages are On the v1 site and will be removed anyway. The others are error pages we don't want indexed.


>  Blocked due to other 4xx issue - 1 page

Done - There was a broken link inside a rule, [fixed](https://github.com/SSWConsulting/SSW.Rules.Content/pull/8813/files)

> 403

Not Done - Both pages are on the old server and can't be fixed.

Server error (5xx)

Done - I added the Tina template pages to robots.txt so google won't attempt to index them.

<br>

I've also created a list of known issues on SharePoint which I've had tested by Penny

I've added a list of known isues in SharePoint
https://sswcom.sharepoint.com/:w:/r/sites/SSWMarketing/_layouts/15/Doc.aspx?sourcedoc=%7B8F4C361D-1664-45FE-B10C-A7EE3E9A30A4%7D&file=Google%20Analytics%20-%20known%20issues%20for%20indexing%20pages.docx&action=default&mobileredirect=true&wdsle=0
